docs/best_practices/integration_checklist.rst

```diff
@@ -60,7 +63,8 @@ Adding Your Tool to the IUC Repository
 - Within that folder, create a corresponding branch. You might name it after the tool.
 - After you have tested your tool and are completely happy with it (per
   previous sections of this document), add your tool and all associated data,
-  then Commit the changes with ``git commit``
+  then Commit the changes with ``git commit``. Finally push your changes
+  to github with ``git push``.
```

Comment:
git learning page is a nice idea and should be included.
`git push origin branchname` works nicely, it's then in the personal branch and you can go to github and click on create PR. I think this is the easiest way, otherwise you need to configure a `remote`.